doxygen/qlib/Doxyfile.in

```diff
@@ -27,20 +31,23 @@ DOXYFILE_ENCODING      = UTF-8
 # The PROJECT_NAME tag is a single word (or a sequence of words surrounded
 # by quotes) that should identify the project.
 
-PROJECT_NAME           = "Qore @QORE_QMOD_FNAME@ Module Reference"
+#PROJECT_NAME           = "Qore @QORE_QMOD_FNAME@ Module Reference"
+PROJECT_NAME           = "Qore {module} Module Reference"
```

Comment:
here and everywhere below the original settings was left commented. Is it necessary? I would remove it and use just the new one, it will be in git history anyway, right?

Reply:
I makes sense leave it as it better solution without qdx but requires rewriting more cmake&autotools&qdx stuff. It is beyond this ticket